src/tables.ts

```diff
@@ -390,47 +390,138 @@ export const Tables = {
       }
     > = {
       C: { index: 0, int_val: 0 },
+      'C+': { index: 0, int_val: 0, accidental: '+' },
```

Comment:
Thanks Rodrigo. Can you move all the microtonals to a separate section to reduce clutter?

```typescript
{
  'C': ...,
  'C#': ...,
  ...
  // Microtonals
  'C+': ...
  ...
}
```

Reply:
Done